### PR TITLE
Adds four new departmental tools

### PR DESCRIPTION
**Pull request by JimKil3**
## What Does This PR Do
Adds departmental tools to each department, each with their own unique descriptions, appearances, and suicide acts!
Also touches up the command multitool's description and gives it in-hand sprites.

Specifically adds the:
- Engineering crowbar
- Security wirecutters
- Research welding tool
- Cargo screwdriver

## Why It's Good For The Game
More variety is better, and completing the trend of departments getting their own tools set by the medical wrench and the command multitool is good.

Service is too spread-out to really get its down departmental tool, and legal is.. not really a department, same with assistants. (Though, in the future, I might add a special assistant toolbelt as well.)

## Images of changes
image

## Testing
Looked at the sprites in-game and suicided a _lot_.

## Changelog
:cl:
add: Added the engineering crowbar, security wirecutters, research welder, and cargo screwdriver
tweak: Tweaked the command multitool's description to read more nicely with more than one suicide
imageadd: Added in-hand sprites for the command multitool
/:cl:

### CODE REVIEW

**Comment by procdrone** [comment]
Time to collect them all and receive superpowers :D

File: code/game/objects/items/tools/multitool.dm
Comment (Sirryan2002) [inline code/game/objects/items/tools/multitool.dm:131]: is this meant to have the base_desc added to it?

File: code/game/objects/items/tools/multitool.dm
Comment (Sirryan2002) [inline code/game/objects/items/tools/multitool.dm:139]:
```suggestion
	victims += user.name
```

File: code/game/objects/items/tools/screwdriver.dm
Comment (Sirryan2002) [inline code/game/objects/items/tools/screwdriver.dm:81]: should be checking for user beforehand

File: code/game/objects/items/tools/screwdriver.dm
Comment (Sirryan2002) [inline code/game/objects/items/tools/screwdriver.dm:84]:
```suggestion
	sleep(2 SECONDS)
```

File: code/game/objects/items/tools/welder.dm
Comment (Sirryan2002) [inline code/game/objects/items/tools/welder.dm:233]:
```suggestion
	sleep(2 SECONDS)
```

File: code/game/objects/items/tools/wirecutters.dm
Comment (Sirryan2002) [inline code/game/objects/items/tools/wirecutters.dm:64]: same here

File: code/game/objects/items/tools/wirecutters.dm
Comment (Sirryan2002) [inline code/game/objects/items/tools/wirecutters.dm:67]:
```suggestion
	sleep(2 SECONDS)
```

**Review by Sirryan2002 — COMMENTED** [review]
couple things, also make sure that none of these tools are more powerful than what's available at round-start

File: code/game/objects/items/tools/multitool.dm
Comment (JimKil3) [reply code/game/objects/items/tools/multitool.dm:131]: No, because if nobody's committed suicide with it, then the description will be the same as the `base_desc`

**Comment by JimKil3** [comment]
> couple things, also make sure that none of these tools are more powerful than what's available at round-start

Currently, these are all somewhat faster than their normal counterparts, but slower than RnD variants. The research welder has an industrial sized tank and less-bright flashing too, but this can all be changed pretty easily

File: code/game/objects/items/tools/multitool.dm
Comment (lewcc) [inline code/game/objects/items/tools/multitool.dm:131]: Why not incorporate `Its screen displays the text` into the base message? It seems constant through all the switch statements.

File: code/game/objects/items/tools/multitool.dm
Comment (lewcc) [inline code/game/objects/items/tools/multitool.dm:136]: You can probably also use the `english_list` proc to make the code a little cleaner here, it's useful for formatting things like this.

File: code/game/objects/items/tools/screwdriver.dm
Comment (lewcc) [inline code/game/objects/items/tools/screwdriver.dm:88]: Maybe make this a visible message?

File: code/game/objects/items/tools/welder.dm
Comment (lewcc) [inline code/game/objects/items/tools/welder.dm:233]: Same here, might be nice to make these visible messages to give some outward flavor.

File: code/game/objects/items/tools/multitool.dm
Comment (JimKil3) [reply code/game/objects/items/tools/multitool.dm:131]: Because the base description is the multitool's description without any suicides, so the screen wouldn't be displaying any text

Comment (lewcc) [reply code/game/objects/items/tools/multitool.dm:131]: Oh, I mean `base_desc`, since that's what gets appended onto here.

**Comment by Probablyjay19** [comment]
Now all we need is a steal objective to collect all the departmental tools infinity stones style

File: code/game/objects/items/tools/crowbar.dm
Comment (S34NW) [inline code/game/objects/items/tools/crowbar.dm:70]:
```suggestion
	toolspeed = 0.5
```
This seems a bit strong for a roundstart tool, although I'm no balance nerd

File: code/game/objects/items/tools/screwdriver.dm
Comment (S34NW) [inline code/game/objects/items/tools/screwdriver.dm:75]: as above

File: code/game/objects/items/tools/welder.dm
Comment (S34NW) [inline code/game/objects/items/tools/welder.dm:223]:
```suggestion
	toolspeed = 0.75
```

as above

File: code/game/objects/items/tools/wirecutters.dm
Comment (S34NW) [inline code/game/objects/items/tools/wirecutters.dm:58]:
```suggestion
	toolspeed = 0.75
```
as above

File: code/game/objects/items/tools/crowbar.dm
Comment (JimKil3) [reply code/game/objects/items/tools/crowbar.dm:70]: This is the speed the large crowbar has by default, but I'm fine with changing it if needed

File: code/game/objects/items/tools/screwdriver.dm
Comment (JimKil3) [reply code/game/objects/items/tools/screwdriver.dm:75]: This doesn't have a different tool to be based on for speed, but it's slower than its RnD counterpart (the hand drill), doesn't have the advantage of being two tools in one, and there's only ever going to be one of these around

File: code/game/objects/items/tools/welder.dm
Comment (JimKil3) [reply code/game/objects/items/tools/welder.dm:223]: Same as the cargo screwdriver, though it has advantages other than the speed as well (the lower flash brightness)

File: code/game/objects/items/tools/wirecutters.dm
Comment (JimKil3) [reply code/game/objects/items/tools/wirecutters.dm:58]: Same as the welder

File: code/game/objects/items/tools/crowbar.dm
Comment (S34NW) [reply code/game/objects/items/tools/crowbar.dm:70]: oh, my bad. Why the hell is the default value for this tool 0.5, what a meme.

File: code/game/objects/items/tools/wrench.dm
Comment (Henri215) [inline code/game/objects/items/tools/wrench.dm:99]:
```suggestion
	// Immobilize stops them from wandering off and dropping the wrench
```

**Review by Sirryan2002 — APPROVED** [review]

**Comment by A-Medster** [comment]
Legal could get a Department Paper bin in the IAA's office. Have it hold like, three to five extra pices of paper?
While maybe Service could get some Service flavor shirts in their lockers, or one set of Service flavored Insulated gloves that randomly appeir in Maintnence or is an item in the vault?

**Review by farie82 — APPROVED** [review]

**Comment by farie82** [comment]
Can be merged after #20659 gets merged and this PR gets updated to master. Be sure to also test after doing the rebase/merge to master

**Comment by SteelSlayer** [comment]
@JimKil3 it appears that something went wrong with one of your recent update commits, and there are now a lot of unrelated map changes and sprite additions. I suspect it was `Merge branch 'departmental-tools' into engineering-crowbar`, but I'm not sure. I recommend that you revert your branch to a point in time before those update commits were pushed. If you need help with this, let me know.

**Review by SteelSlayer — APPROVED** [review]

**Comment by SteelSlayer** [comment]
Everything looks good now!

**Comment by tipoima** [comment]
>Service is too spread-out to really get its down departmental tool
Service already has a departmental tool. His Grace is the Service Toolbox.

**Pull request merged**
